projects/systelab-components/src/lib/spinner/spinner-with-decinals.component.spec.ts

```diff
@@ -0,0 +1,161 @@
+import { Component } from '@angular/core';
+import { async, ComponentFixture, TestBed } from '@angular/core/testing';
+import { TouchSpinValues } from './touch.spin-values';
+import { BrowserModule, By } from '@angular/platform-browser';
+import { BrowserAnimationsModule } from '@angular/platform-browser/animations';
+import { FormsModule } from '@angular/forms';
+import { DragDropModule } from '@angular/cdk/drag-drop';
+import { OverlayModule } from '@angular/cdk/overlay';
+import { TreeModule } from 'primeng/tree';
+import { HttpClientModule } from '@angular/common/http';
+import { TouchspinComponent } from './spinner.component';
+import { DatepickerTestComponent } from '../datepicker/datepicker.component.spec';
+
+@Component({
+	selector: 'systelab-spinner-test',
+	template: `
+                <div>
+                    <systelab-spinner [spinValues]="values" (change)="doValueChange()"></systelab-spinner>
+                    <label class="label-value">{{values.value}}</label>
+                </div>
+	          `,
+	styles:   []
+})
+export class SpinnerTestComponent {
+	public values = new TouchSpinValues(34.12345, 1.00001, 100.001, 1.12345, true, 5);
+
+	public doValueChange() {
+	}
+}
+
+describe('Systelab Spinner with decimals', () => {
+	let fixture: ComponentFixture<SpinnerTestComponent>;
+
+	beforeEach(async(() => {
+		TestBed.configureTestingModule({
+			imports:      [BrowserModule,
+				BrowserAnimationsModule,
+				FormsModule,
+				DragDropModule,
+				OverlayModule,
+				TreeModule,
+				HttpClientModule],
+			declarations: [TouchspinComponent, SpinnerTestComponent]
+		}).compileComponents();
+	}));
+
+	beforeEach(() => {
+		fixture = TestBed.createComponent(SpinnerTestComponent);
+		fixture.detectChanges();
+	});
+
+	it('should instantiate', () => {
```

Comment:
That is definitely a good practice for the e2e tests, specially because it fits very well with the Allure reports. However, I was not aware that it was a recommendation also for the unit tests. 
Maybe we should discuss this at some point to have a uniform convention and prevent mixing both ways.

Reply:
Let's talk about it, but, for units, it is the standard set by angular cli.